src/WebJobs.Script.Grpc/Abstractions/WorkerCreateContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Rpc
+{
+    public class WorkerCreateContext
+    {
+        public Uri ServerUri { get; set; }
+
+        public WorkerConfig WorkerConfig { get; set; }
```

Comment:
>WorkerConfig [](start = 15, length = 12)

What's the difference between a WorkerCreateContext and WorkerConfig? Are they 1:1?

Reply:
WorkerConfig is currently a description of a language worker process (how to start node, what arguments to pass, what functions are supported). Long term, I want this to be owned by the language worker and live in that nuget.

WorkerCreateContext is the information for a single call to create a generic worker on the host side.